src/main/java/screens/GraphScene.java

```diff
@@ -19,32 +19,59 @@
  */
  /*package*/ class GraphScene extends Pane {
 
+    /**
+     * State for handling center queries.
+     */
     private final INodeHandler center;
+
+    /**
+     * State for handling information display queries.
+     */
     private final INodeHandler info;
+
+    /**
+     * The current state for node events.
+     */
     private INodeHandler state;
-     /**
+
+    /**
      * Event handler for when a node or edge is clicked.
      */
      private EventHandler<MouseEvent> click = event -> {
 
         if (event.getSource() instanceof DrawNode) {
+            /**
+             * DrawNode object that is linked to the Node object.
+             */
             DrawNode rect = (DrawNode) (event.getSource());
             state.handle(rect);
         } else if (event.getSource() instanceof Line) {
+            /**
+             * Line object.
+             */
             Line l = (Line) (event.getSource());
+            /**
+             * Id of line object.
+             */
             String edgeNodes = l.getId();
-            System.out.println(edgeNodes);
             Window.getInfoScreen().getTextArea().appendText("Edge from node " + edgeNodes.substring(0, edgeNodes.indexOf("-")) + " to " + edgeNodes.substring(edgeNodes.indexOf("-") + 1, edgeNodes.length()) + "\n");
         }
      };
 
-
+    /**
+     * GraphScene pane constructor.
+     */
      /*package*/ GraphScene() {
```

Comment:
This modifier is not ignored by the compiler. This prevents this method from being accessed by classes outside this package.